open_inside_file_iter.chpl:1: In iterator 'split':
open_inside_file_iter.chpl:2: error: unresolved call 'file.open(string, iomode, int(32), iostyle)'
$CHPL_HOME/modules/standard/IO.chpl:527: note: candidates are: file.open()
$CHPL_HOME/modules/standard/IO.chpl:711: note:                 open(out error: syserr, path: string = "", mode: iomode, hints: iohints = IOHINT_NONE, style: iostyle = defaultIOStyle(), url: string = "")
$CHPL_HOME/modules/standard/IO.chpl:764: note:                 open(path: string = "", mode: iomode, hints: iohints = IOHINT_NONE, style: iostyle = defaultIOStyle(), url: string = "")
